etc/DependencyInstaller.sh

```diff
@@ -132,6 +132,15 @@ _installCommonDev() {
 
     cd "${lastDir}"
     rm -rf "${baseDir}"
+
+    if [[ ! -z ${PREFIX} ]]; then
```

Comment:
Does env.sh and setup_env.sh in ORFS need to be updated?

Reply:
"need to be" - no. This is just a convenience addition for installations making use of `-prefix`. It doesn't change any behavior with current scripts, although after merging they could make use if they wanted to.

Reply:
So update env.sh and dev_env.sh to use a single source of truth instead of duplicating how to set up paths?

Reply:
Sure

Reply:
@rovinski you asked for a review but this item seems unresolved.  Is there more to do yet?

Reply:
There would be nothing to resolve here, only in ORFS after this is merged.